src/generator/AutoRest.Ruby.Azure/Templates/AzureModelTemplate.cshtml

```diff
@@ -0,0 +1,69 @@
+﻿@using System
```

Comment:
The entire file seems like exact copy of `ModelTemplate` from `Ruby` generator except line number 49. I'd definitely recommend  not repeating as long as possible.